### ISSUE TITLE
Silent option for commands

### ISSUE THREAD

**Issue by MageAtoru** [nulli0n/GoldenCrates#98]
Would it be possible to get a "silent option" for giving crate keys
`/goldencrates givekey PlayerName common 1 -silent`

- when "-silent" is used target player receives the key but not the message

This option would help greatly for vote rewards _(but also some other instances like custom mob/boss drops)_ where displaying the message to player that he has gotten a key is unnecessary since he already got told so by the vote plugin...

 Basically I just want to reduce the clutter in chat for player when it comes to voting...
![1630137605_1a4e5b6 1](https://user-images.githubusercontent.com/23415894/131211044-2e82f191-b61f-4cac-987b-62596b8b96d3.png)

**Comment by Chiquis2005** [comment]
+1
